src/components/menu/menu-item/menu-item.component.ts

```diff
@@ -127,7 +138,7 @@ export class MenuItemComponent implements OnInit, OnDestroy, FocusableOption {
     @HostListener('click', ['$event'])
     @HostListener('keydown.enter', ['$event'])
     _onClick(event: MouseEvent | KeyboardEvent): void {
-        if (!this.disabled) {
+        if (!this.disabled && this.closeOnSelect) {
             this.onClick$.next(isKeyboardTrigger(event) ? 'keyboard' : 'mouse');
             this.activate.emit(event);
```

Comment:
The above could be the cause of this, we are stopping the activate from being emitted here. Maybe we could have another if statement just for `closeOnSelect` which stops the `this.onClick$.next` but keeps the `this.closeOnSelect`?